### ISSUE TITLE
[ChoiceGroup][Firefox][High Contrast] Inner circle for selected option is not visible

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1955]
- __Browser and OS versions__: Firefox

#### Describe the issue:

Choice group does not show inner circle for selected option in firefox when high contrast is enabled.

#### Actual behavior:

Only outer ring is visible.

#### Expected behavior:

Selected option should have an inner circle to highlight choice

### If applicable, please provide a codepen repro:
Scenario can be repro'd at https://dev.office.com/fabric#/components/choicegroup

![image](https://user-images.githubusercontent.com/11937034/26928170-aae18942-4c72-11e7-9f41-be4add25be46.png)
